file-formats/dobj/type/zif_aff_dobj_v1.intf.abap

```diff
@@ -0,0 +1,61 @@
+INTERFACE zif_aff_dobj_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Structure Definition</p>
+    "! Structure Definition
+    BEGIN OF ty_structure_definition,
+      "! <p class="shorttext">Structure Definition</p>
+      "! Parent Table
+      "! $required
+      record_no       TYPE n LENGTH 4,
```

Comment:
Is the description correct?

Is the record_no used for sorting the entries only? Or is it really a key field?

Reply:
It is modified based on the review comments and as such is maintained same as in On-Premise transactions. It is a key field.